File: config/src/main/java/org/wcl/mfn/config/ui/mvc/ViewConfig.java
package org.wcl.mfn.config.ui.mvc;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.context.annotation.*;

@Configuration
@PropertySource({"ui/mvc/view/view.properties"})
public class ViewConfig {
    @Value("${view.home}")
    private String homeView;

    @Value("${view.tools.contract-calculator}")
    private String contractCalculatorView;

    public String homeView() {
        return homeView;
    }

    public String contractCalculatorView() {
        return contractCalculatorView;
    }
}

File: config/src/main/kotlin/org/wcl/mfn/config/ui/mvc/ViewConfig.kt
package org.wcl.mfn.config.ui.mvc

import org.springframework.beans.factory.annotation.Value
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.PropertySource

@Configuration
@PropertySource("classpath:ui/mvc/view.properties")
open class ViewConfig {
    @Value("\${view.home}")
    private val homeView: String? = null

    @Value("\${view.tools.contract-calculator}")
    private val contractCalculatorView: String? = null

    fun homeView(): String? {
        return homeView
    }

    fun contractCalculatorView(): String? {
        return contractCalculatorView
    }
}
